recipe/meta.yaml

```diff
@@ -1,42 +1,75 @@
 {% set name = "pytorch-forecasting" %}
-{% set version = "0.10.2" %}
+{% set version = "0.10.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/pytorch_forecasting/pytorch_forecasting-{{ version }}.tar.gz
-  sha256: 069af34c17eea2edf5c11c79931097427b88cef8a862457818a03464260e5534
+  url: https://github.com/jdb78/pytorch-forecasting/archive/refs/tags/v0.10.3.tar.gz
```

Comment:
Thanks for pointing to this URL. I replace PyPi url to get the `tests` folder (pypi https://files.pythonhosted.org/packages/02/f0/92eb7e55f665de688120590bcee95b6704f8c6c07ae07d1255ecf734997e/ hasn't it pytorch_forecasting-0.10.3.tar.gz) for further testing the package.

Reply:
this should be discussed upstream, probably sending a PR to include tests in sdist. github tag tarball misses PKG-INFO, that is probably why dynamic versioning fails. I'd say fetch both files and copy just the tests from github tag tarball until that's fixed upstream. it is probably better than patching pyproject.toml

Reply:
dynamic versioning fails even if I use PyPi url.

Reply:
this is wrong. the sdist file won't work as is, even with PKG-INFO file included. The source tarballs from upstream are seriously broken. I'd suggest discussing it with maintainer, and applying the same patches here until they are released upstream, instead of carrying custom patches.

Reply:
> dynamic versioning fails even if I use PyPi url.

yes, I found that out just now. this must be discussed further with upstream

Reply:
@rafaelmartins Do you think this is blocking us to deliver the package to the destination channel? Because I think if we can build and test the package that it's working we can go ahead

Reply:
This way the poetry plugin will detect the version without the need of any patching. This is the file that is missing in the sdist tarball

Reply:
Thanks. That's much better, and it's great to learn something new